packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -39,6 +40,16 @@ export class ChannelService {
     );
   }
 
+  public streamCommandOutputWithoutColor(execution: CommandExecution) {
```

Comment:
These changes should probably be done to `streamCommandOutput` and have it work the same way for every command that outputs using the channel service. Does adding `stripAnsi` to `streamCommandOutput` break other commands ?

Reply:
I haven't tried that out because I think adding it as an additional method is less risky since a lot of commands use `streamCommandOutput`. I could add it to `streamCommandOutput` if you like.

Reply:
I think it should be safe to add it since the other commands process cli output for things like deploy, queries, etc which do not contain ansi characters unless explicitly asked for. @AnanyaJha might know more about this since she had to add it to one of the rewritten Apex commands.

Reply:
Updated - integration tests and sanity check looks good